jupyterhub/spawner.py

```diff
@@ -627,6 +627,10 @@ def my_hook(spawner):
         """
     ).tag(config=True)
 
+    def handle_auth_state(self, auth_state):
+        """Pass optional auth_state to Spawner."""
+        pass
```

Comment:
Should there perhaps be a configurable auth_state_hook so this can be used via config without subclassing?